### PR TITLE
Set configurations from loaded .ini file

### PR DESCRIPTION
**Pull request by michalrudolf**
A3IDES-279
A3IDES-280

Set configurations from loaded .ini file through thread-safe netapi function.

(Previous version of this PL(https://github.com/prusa3d/Prusa-Firmware-Buddy/pull/5) needed to be closed due to repository publishing. My private fork was no longer connected)

### CODE REVIEW

File: src/gui/screen_lan_settings.c [hunk 1]
```diff
@@ -5,11 +5,13 @@
  *      Author: Migi
  */
 
+#define LAN_CONFIG_DEVELOPMENT
```
Comment (joshymjose) [inline src/gui/screen_lan_settings.c]: you may remove this macro

File: src/gui/screen_lan_settings.c [hunk 2]
```diff
@@ -125,18 +127,17 @@ static void screen_lan_settings_init(screen_t *screen) {
     //============== DECLARE VARIABLES ================
 
     plan_str = (char *)gui_malloc(150 * sizeof(char));
-    char mac_addr_str[18];
 
     //============= FILL VARIABLES ============
 
-    _parse_MAC_addr(mac_addr_str);
+    _parse_MAC_addr(plsd->mac_addr_str);
```
Comment (joshymjose) [inline src/gui/screen_lan_settings.c]: why "_function_name" and "function_name"? why some functions starts with under score?

File: src/gui/screen_lan_settings.h [hunk 3]
```diff
@@ -22,6 +22,7 @@ typedef struct {
     menu_item_t *items;
 
     window_text_t text;
+    char mac_addr_str[18];
```
Comment (joshymjose) [inline src/gui/screen_lan_settings.h]: magic number "18"  use maco to define it

File: src/gui/screen_lan_settings.c (hunk 2 above)
Comment (michalrudolf) [reply src/gui/screen_lan_settings.c]: These are just "private" functions, that are not included in header and are used only in this source file.

**Pull request merged**